package.json

```diff
@@ -12,6 +12,7 @@
     "prettier": "prettier . --write",
     "prettier:check": "prettier . --check",
     "lint": "eslint amt/site/static/ts/",
+    "pre-commit": "npm run lint && npm run prettier",
```

Comment:
What does this line do?

Reply:
This is not the same as the python pre-commit tool (not doesnt het picked up), right? This is for husky? Should/could we add husky to the python pre-commit so that it gets picked up? Or is that weird?

Reply:
Or maybe we skip husky altogether and using pre-commit directly to run these nmp linter/pretiers?

Reply:
Yeah this is the husky pre-commit one, and will run during pre-commit automagically, I will find a way to run it through the python pre-commit to have everything in one place (I will at least not use the native python pre-commit to run eslint as we could get again different packages between this one and the package.json)